rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -100,18 +108,18 @@ def mock_query_price(from_asset, to_asset):
         assert to_asset.identifier == 'USD'
         nonlocal call_count
         if call_count == 0:
-            price = {'USD': '1'}
+            price = Price(FVal('1'))
         elif call_count in (1, 2):
-            price = {'USD': '2'}
+            price = Price(FVal('2'))
         else:
             raise AssertionError('Called too many times for this test')
 
         call_count += 1
         return price
 
     cc_patch = patch.object(
-        inquirer._cryptocompare,
-        'query_endpoint_price',
+        inquirer._oracle_instances[0][1],
```

Comment:
if the default order changes this test will fail by doing this. Why are you doing this? `_cryptocompare` is the object attribute the test is patching here. This is not changing.